app/views/categories/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= stylesheet_link_tag 'category_index' %>
+<% content_for :page_name, "Transactions" %>
+
+<%= render 'helper/navbar' %>
+<div class="px-4 d-flex justify-content-between align-items-center top-card">
+  <h2 class="second-color"><%= @category.name %></h2>
+  <h2 class="second-color">$ <%= @category.total %></h2>
+</div>
+<div class="container-fluid">
+  <% @category.buys.order(created_at: :desc).each do |buy| %>
+    <div class="card mt-3 px-3">
+      <div class="card-body d-flex justify-content-between">
+        <div class="d-flex align-items-center">
+          <h2 class="text-decoration-none second-color"><%= buy.name %></h2>
+        </div>
+        <h3 class="second-color align-self-center" >$ <%= buy.amount %></h3>
+      </div>
+    </div>
+  <% end %>
+
+  <div class="d-flex justify-content-center mt-4">
+    <%= link_to 'Add New Transaction', new_buy_path, id: 'add-transaction', class: 'btn btn-primary', style: "width: 100%;" %>
+  </div>
```

Comment:
- Also, please ensure that the `Add New Transaction` button follows the color code i.e `#5fb523` and is aligned to the bottom of the page as I pointed out for the button in the category#index page. Please fix this 👍 👍

Design template:
![wq4rq2w4t](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/d50aaea8-d317-4843-b747-9ddcd10fda51)

Reply:
done